tests/installation/partitioning_filesystem.pm

```diff
@@ -23,16 +24,31 @@ sub run {
     assert_and_click 'edit-proposal-settings';
 
     if (get_var('PARTITIONING_WARNINGS')) {
-        assert_screen 'proposal-will-overwrite-manual-changes';
-        send_key 'alt-y';
+        if (is_storage_ng && !check_screen 'proposal-will-overwrite-manual-changes', 0) {
```

Comment:
because the previous action is `assert_and_click 'edit-proposal-settings'` I would not be so sure that the system already reached steady state yet. I am not sure the check_screen with 0s timeout will work here. So in case there *is* a popup but it's not showing up in time we record the soft failure - which is not that bad as long as the bug is still open - but then we are most likely stuck because we never press the next key. The only alternative I can think of right now is to clone the next screens needle into a workaround needle with that tag.